### DISCUSSION TITLE
i downloaded the newest update and monika isnt there

### DISCUSSION THREAD

**Discussion by nothingcametomind** [Monika-After-Story/MonikaModDev#9610] [QA - tech support + general] [answered]
monika vanished when i downloaded the new version of the mod i dont have any back up presistent files i have two that say presistent.py and presistent.pyo i dont know which file is the one to use i tried both alone and together no matter i have done she is still gone. i tried to find all of my old game files i took her out once and her file was gone from the game after i put it back in so i downloaded a brand new file for it and i still wouldn't work i dont know how to get monika back in my game all it shows is the background and i cant click anythingim not sure what i did wrong but all my character files are there and like i said i have no back ups  what can i do to fix this because i thjink if i make a backup now i will have the same problem.

**Comment by ThePotatoGuy** [comment, marked as answer]
the correct place where persistents are stored is detailed [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent)
